### PR TITLE
Rename odl to uniconfig

### PR DESCRIPTION
**Pull request by jrozsa**
Signed-off-by: Jozef Rózsa <jrozsa@frinx.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # FRINX-machine
 The project is a containerized package of: 
 
-* [FRINX ODL]
+* [FRINX Uniconfig]
```
Comment (MartinSunal) [inline README.md]: UniConfig

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The project is a containerized package of:
 * 16GB and 4 CPU
 * [Docker](https://www.docker.com/)
 * [Docker Compose](https://github.com/docker/compose)
-* License for FRINX ODL (you can find a trial license in the "Installation Guide" section below)
+* License for FRINX Uniconf (you can find a trial license in the "Installation Guide" section below)
```
Comment (MartinSunal) [inline README.md]: UniConfig

File: README.md [hunk 3]
```diff
@@ -141,7 +141,7 @@ sudo docker rm [service]
 sudo docker-compose up -d [service]
 ```
 
-[FRINX ODL]: <https://frinx.io/odl_distribution>
+[FRINX UNICONFIG]: <https://frinx.io/odl_distribution>
```
Comment (MartinSunal) [inline README.md]: UniConfig

**Comment by MartinSunal** [comment]
pls, rename "odl" folder to "uniconfig"

File: README.md (hunk 1 above)
Comment (jrozsa) [reply README.md]: Changed it.

File: README.md (hunk 2 above)
Comment (jrozsa) [reply README.md]: Changed it.

File: README.md (hunk 3 above)
Comment (jrozsa) [reply README.md]: Changed it.

**Pull request merged**